platform/viewer/public/theme/theme.json5

```diff
@@ -0,0 +1,80 @@
+{
+  ThemeProtocols: {
+    protocols: [
+      {
+        id: '1x2',
```

Comment:
Note how this is somewhat easier to define - it has a bunch of stuff defaulted or not required.